tests/Doctrine/Tests/DBAL/ConnectionTest.php

```diff
@@ -405,7 +405,7 @@ public function testConnectionIsClosed()
     {
         $this->_conn->close();
 
-        $this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
+        //$this->setExpectedException('Doctrine\\DBAL\\Exception\\DriverException');
 
         $this->_conn->quoteIdentifier('Bug');
```

Comment:
Actually, `$this->platform` will *never* fail being detected: https://github.com/doctrine/dbal/blob/master/lib/Doctrine/DBAL/Connection.php#L386. 

But more importantly, this test was added to test this change: https://github.com/doctrine/dbal/pull/688/files#diff-7b1e94fb7dd408985f7b7973330dc2edL600 (you can see this referenced from #691 where this change came from).

Basically, the test was to check to make sure that if the connection is closed, we don't end up getting any "property doesn't exist" type errors. I'm not sure that's something that can be tested, and I can't see how this test was ever valid. So, either this test should be removed (because it's totally not testing what it was intended to test) or I'm completely missing the purpose of this test. Both are possible :)